obs/export.py

```diff
@@ -139,15 +139,10 @@ def checkForStandardKeysJSON():
     if 'checkinglevel' not in body_json.keys(): body_json['checkinglevel'] = args.checkinglevel
 
 def writeFile(outfile, p):
-    makeDir(outfile.rpartition('/')[0])
```

Comment:
@jag3773 I explained the logic for that in the commit message. It kills the script in some cases and rather than fix it I decided it would be better to keep the this sort of i/o handling in one place: the shell scripts that call this are a more appropriate place to handle making dirs.